prow/gerrit/adapter/adapter.go

```diff
@@ -50,10 +50,14 @@ const (
 )
 
 var gerritMetrics = struct {
-	processingResults     *prometheus.CounterVec
-	triggerLatency        *prometheus.HistogramVec
-	changeProcessDuration *prometheus.HistogramVec
-	changeSyncDuration    *prometheus.HistogramVec
+	processingResults           *prometheus.CounterVec
+	inrepoconfigFailures        *prometheus.CounterVec
+	triggerLatency              *prometheus.HistogramVec
+	triggerHelpLatency          *prometheus.HistogramVec
+	changeProcessDuration       *prometheus.HistogramVec
+	processSingleChangeDuration *prometheus.HistogramVec
```

Comment:
I wonder if this is close to the `changeProcessDuration` divided by `avg number of changes per host`, should we add that metric too?

Reply:
I don't think so? We'd get the granularity we want with `processSingleChangeDuration` so that the proposed new metric you described would be redundant.

Reply:
I think having the average number of changes would be more useful than average processing time per change.

Reply:
> average processing time per change

We're not measuring averages. The `processSingleChangeDuration` metric is a histogram.

> I think having the average number of changes would be more useful

What does it matter if we process 10 changes on average per repo (per day, for example) vs 100, or 1000? The busier a repo is, the more changes it will have that need to be processed by our Gerrit adapter, right?